src/components/Tabs.js

```diff
@@ -159,31 +190,19 @@ class Tabs extends React.Component {
           color: StyleConstants.Colors.CHARCOAL
         }
       },
-      activeTab: {
+      activeTab: Object.assign({
         cursor: 'default',
         color: this.props.brandColor,
         borderBottom: '2px solid ' + this.props.brandColor,
 
         ':hover': {
           color: this.props.brandColor
         }
-      },
-      menuWrapper: {
-        alignItems: 'center',
-        boxSizing: 'border-box',
-        color: this.props.brandColor,
-        lineHeight: '20px',
-        fontSize: StyleConstants.FontSizes.MEDIUM,
-        fontStyle: StyleConstants.Fonts.SEMIBOLD
-      },
-      tabsContainer: {
-        display: 'flex',
-        justifyContent: this.props.alignment === 'left' ? 'flex-start' : 'center',
-        borderBottom: this.props.showBottomBorder ? '1px solid ' + StyleConstants.Colors.FOG : 'none',
-        width: '100%'
-      }
+      }, this.props.activeTabStyles)
     };
-  };
+  }
 }
 
-module.exports = Radium(Tabs);
+const Tab = Radium(TabWithoutRadium);
```

Comment:
The `Tabs` component uses `Tab` which needs Radium for the hover styles. I named it `TabWithoutRadium` to communicate that it still needs to be wrapped with the Radium higher order component before it can be used as the `Tab` component.

Do you think a different name would make it clearer?